docs/en/lfs.md

```diff
@@ -1,205 +1,144 @@
+I have found that moving code into LFS has changed my coding style, as I tend to use larger modules and I don't worry about in-memory code size. This make it a lot easier to adopt a 'keep it simple stupid' coding style, so my ESP Lua code now looks more similar to host-based Lua code.  Lua code can still be loaded from SPIFFS, so you still have the option to keep code under test  in SPIFFS, and only move modules into LFS once they are stable.  If you use `require` to load modules then this can all be handled transparently by the require loader.
 
-Here is the code fragment that I use in my `_init` module to do this magic:
+#### Accessing LFS functions and loading LFS modules
 
-```Lua
-do
-  local index = node.flashindex
-  -- Setup the LFS object
-  local lfs_t = {
-    __index = function(_, name)
-        local fn_ut, ba, ma, size, modules = index(name)
-        if not ba then
-          return fn_ut
-        elseif name == '_time' then
-          return fn_ut
-        elseif name == '_config' then
-          local fs_ma, fs_size = file.fscfg()
-          return {ba, ma, fs_ma, size, fs_size}
-        elseif name == '_list' then
-          return modules
-        else
-          return nil
-        end
-            end,
-    __newindex = function(_, name, value)
-        error("LFS is readonly. Invalid write to LFS." .. name, 2)
-      end
-  }
-  rawset(getfenv(),'LFS', setmetatable(lfs_t,lfs_t))
-  -- And add LFS to the require path list
-  local function loader_flash(module)
-    local fn, ba = index(module)
-    return ba and "Module not in LFS" or fn
-  end
-  table.insert(package.loaders, loader_flash)
-
-end
-```
+See [lua_examples/lfs/_init.lua](https://github.com/nodemcu/nodemcu-firmware/tree/dev/lua_examples/lfs/_init.lua) for the code that I use in my `_init` module to do create a simple access API for LFS.  There are two parts to this.
 
-This code is also conveniently packaged in [lua_examples/lfs/_init.lua](../../lua_examples/lfs/_init.lua).
+The first sets up an LFS table in `_G` with the `__index` and `__newindex` metamethods. The main purpose of the `__index()` is to resolve any names against the LFS using a `node.flashindex()` call, so that `LFS.someFunc(params)` does exactly what you would expect it to do: this will call `someFunc` with the specified parameters, if it exists in in the LFS.  The LFS properties `_time`, `_config` and `_list` can be used to access the other LFS metadata that you need.  See the code to understand what they do, but `LFS._list` is the array of all module names in the LFS.
```

Comment:
"an `LFS` table in `_G`" ?

Reply:
How would you describe 
````Lua
local G=getfenv()
G.LFS = setmetatable(lfs_t,lfs_t)
```
How about "The first sets up a table in the global variable `LFS` with the `__index` and `__newindex` metamethods"?

Reply:
Oh, sorry for being overly short.  I just meant to put backticks around LFS: `LFS`.  I think the prose is fine.